src/eva/core/models/modules/head.py

```diff
@@ -25,7 +25,7 @@ class HeadModule(module.ModelModule):
 
     def __init__(
         self,
-        head: Dict[str, Any] | MODEL_TYPE,
+        head: MODEL_TYPE | Dict[str, Any],
```

Comment:
we shouldn't change the order as we want to prioritise `Dict` so the model to initialized at `configure_model` step, so in every run to have different set of init params

Reply:
Reverted this change now, I wasn't aware that this might have a functional impactt.